src/User.java

```diff
@@ -126,31 +126,33 @@ public BookGroup getBookGroup(int bookGroupID){
             }
         }
         //should never happen
-        return new BookGroup(-1, -1, "", "", "", "", -1, -1);
+        return new BookGroup(-1, -1, "", "", "", "", -1, -1, false, false);
     }
 
     public void addToCurrentlyReading(CurrentBook bookToAdd){
         currentlyReading.add(bookToAdd);
         addToCurrentlyReadingCsv(bookToAdd);
     }
 
-    public boolean checkIfInCurrentBooks(int idToCheck){
+    //NEW SPRINT3 - CHANGED TO RETURN THE INDEX NUMBER OF BOOK IN CURRENTLYREADING
```

Comment:
not sure if java does negative indexing but for the purpose of this -1 means not in the array